### ISSUE TITLE
 What rnn.output_size is referring to exactly?

### ISSUE THREAD

**Issue by katyakuda** [IBM/pytorch-seq2seq#158]
This line is not compatible with 0.4: 
https://github.com/IBM/pytorch-seq2seq/blob/f146087a9a271e9b50f46561e090324764b081fb/seq2seq/models/TopKDecoder.py#L79
The rnn output for lstm is in a shape `(seq_len, batch, num_directions * hidden_size)`
Could you please specify what `output_size` is referring to exactly?

**Comment by pskrunner14** [comment]
From what I gather it's the size of the vocabulary or to be exact the number of probabilities the softmax of the decoder rnn outputs with each forward pass.

**Comment by pskrunner14** [comment]
@kylegao91 #158 can be closed.
